proto/src/determined/api/v1/experiment.proto

```diff
@@ -308,7 +309,18 @@ message CreateExperimentRequest {
   bool activate = 5;
   // Project id to contain the experiment.
   int32 project_id = 6;
+  // Template to use for the experiment.
+  string template = 7;
+  // Git remote at submission time.
+  string git_remote = 8;
+  // Git commit at submission time.
+  string git_commit = 9;
+  // Git committer at submission time.
+  string git_committer = 10;
+  // Git commit date at submission time.
```

Comment:
is it possible to make these all `optional`, and get rid of `emptyStrToNil` conversion?

Reply:
is it?  I can do that